app/assets/javascripts/protocols/steps.js.erb

```diff
@@ -104,14 +103,14 @@ function applyEditCallBack() {
     toggleButtons(false);
     initializeCheckboxSorting();
 
+    TinyMCE.init();
```

Comment:
It's probably not possible to initialize this completely automatically on each field that is `$(textarea.tinymce)`, without explicitly calling this JS, right?